src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -324,18 +323,11 @@ private void validateHearingRequest(HearingRequest createHearingRequest) {
     private void validateHearingRequest(UpdateHearingRequest hearingRequest) {
         validateHearingRequestDetails(hearingRequest);
         validateHearingDetails(hearingRequest.getHearingDetails());
-        validateAmendReasonCodeForUpdate(hearingRequest.getHearingDetails().getAmendReasonCode());
         if (hearingRequest.getPartyDetails() != null) {
             validatePartyDetails(hearingRequest.getPartyDetails());
         }
     }
 
-    private void validateAmendReasonCodeForUpdate(String amendReasonCode) {
```

Comment:
I have removed this as it can easily be represented as a JSR-303 annotation ?

Reply:
Reinstated in https://github.com/hmcts/hmc-cft-hearing-service/pull/242/commits/9060477b00566402258396dd97f0ec1ca4e20604